portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,82 @@
+package com.google.sps.data;
+
+import java.util.Date;
+
+/**
+ * Class representing a comment.
+ */
+public class Comment {
+
+  /*The comment left. */
+  private String comment;
+
+  /*The name of who left the comment. */
+  private String commenter;
+ 
+  /*The time and date when the comment was submitted.*/
```

Comment:
If I'm looking for things to point out, I'd add a space here. I'm getting pretty nit-picky though. :cat: